Makefile

```diff
@@ -7,9 +7,9 @@ IS_DOCKER_INSTALLED = $(shell which docker >> /dev/null 2>&1; echo $$?)
 PACKAGES = $(shell go list ./... | grep -v '/vendor/')
 
 # docker info
-DOCKER_REPO ?= litmuschaos
+DOCKER_REPO ?= shubh214
 DOCKER_IMAGE ?= chaos-operator
-DOCKER_TAG ?= ci
+DOCKER_TAG ?= 2
```

Comment:
@shubhamchaudhary Can you follow the approach like, Build the with tag `commit-hash` and `latest`, Push the `commit-hash` based image to docker hub after that run the BDD test if everything goes good then push the latest tag image also

Reply:
Which repo should I use to build the chaos-operator image?. If I will use repo:chaos-operator then what will be the dockerhub credentials.

Reply:
We have to set the docker hub credential in the circle ci configuration not in the code, For credentials, check with karthik